src/CattyTests/PlayerEngine/Functions/Text/RegularFuntionTest.swift

```diff
@@ -0,0 +1,90 @@
+/**
+ *  Copyright (C) 2010-2020 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+
+import XCTest
+
+@testable import Pocket_Code
+
+class RegularFunctionTest: XCTestCase {
+
+    var function: RegularExpressionFunction!
+
+    override func setUp() {
+        super.setUp()
+        function = RegularExpressionFunction()
+    }
+
+    override func tearDown() {
+        function = nil
+        super.tearDown()
+    }
+
+    func testDefaultValue() {
+        XCTAssertEqual(type(of: function).defaultValue, function.value(firstParameter: "([0-9]{10})" as AnyObject, secondParameter: "I am a panda" as AnyObject))
+    }
+
+    func testValue() {
```

Comment:
Your test looks good. Could you please also document the behavior in case of an invalid regular expression showing an error message (e.g. add another test method "testValueWithInvalidExpression" or similar)?